packages/health-gorilla-core/src/lab-order.ts

```diff
@@ -264,7 +264,7 @@ export function createLabOrderBundle(inputs: PartialLabOrderInputs): Bundle {
       performer: [performerReference],
       category: [{ coding: [{ system: SNOMED, code: '103693007', display: 'Diagnostic procedure' }] }],
       code: { coding: [test] },
-      priority: metadata?.priority || 'routine',
+      priority: metadata?.priority ?? 'routine',
```

Comment:
This also changes behavior if `priority` is an empty string. Probably not a big deal for typescript, but does allow users to pass empty string and it will not assign default value of `routine`

Reply:
Eh....  That comes from the `testMetadata` lookup table, so someone would have had to construct a full table and specifically set `priority` to empty string.

We can revert.  I think Sonar is too aggressive with this rule, and it does lead to pushing us to less desirable behavior.  I wish it didn't ding us for that.  I think this case is probably fine though.

Reply:
yeah now that I look at this you would have to be in some pretty weird territory